### PR TITLE
OKTA-591733 Add maintenance token usage section to custom authenticator guide

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update Access token management section to add "Add Maintenance token configuration and usage" section to custom authenticator guide (for both iOS and Android). Also add Postman collection example.
- **Is this PR related to a Monolith release?** No

### Preview:
https://641c9eb002e2a006f325ff99--reverent-murdock-829d24.netlify.app/docs/guides/authenticators-custom-authenticator/android/main/

image

### Resolves:

* [OKTA-591733](https://oktainc.atlassian.net/browse/OKTA-591733)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/b637a15148565f10ea7010e3272e5b0f5d57572a/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0adb0091-1eff-465a-a4cb-69bff29ea77a) | 53  | 73  | 47  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.12ee5f71-f860-4fa3-9062-de596b7fd930?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/ad226a0eb1afa597252214bf4dc2b359130de6ef/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/fa65e00b-b1ea-4b50-9448-638dd79d0a72) | 47  | 74  | 47  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6fba9257-0370-4c1f-80c1-1cb4a0ba032a?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/e56572228d5b469ccf06449d8944f721963cb46b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8b3ac64c-e2bf-47e5-a07f-96393c54c3d3) | 52  | 74  | 50  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.0c4fde8c-9f21-4369-ab82-47bb46c3c639?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/c9e7e76913c7005ca132d8ef4d1f46924f3a3925/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [76](https://okta.acrolinx.cloud/api/v1/checking/scorecards/55a367ec-4a82-40b1-aacd-fe74ae1f93f3) | 51  | 74  | 52  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.fde113e2-57f0-4e44-8a77-2e697a5c0466?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/8982597f91e5a970f3184e55498ee301e8d3bf6b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/a6861899-004b-402d-9e2c-b383fb984c38) | 35  | 93  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/8982597f91e5a970f3184e55498ee301e8d3bf6b/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/210b5e16-ae2e-4aea-a5a6-721660968f8e) | 49  | 73  | 52  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.625e29e8-6a93-4863-9340-7d47b84e1c19?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/92f1d085aba7379028ff03c49c6f4ae8dc56ad76/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/10c9f216-8abe-4b6c-93d8-1ad96c459a37) | 35  | 93  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/92f1d085aba7379028ff03c49c6f4ae8dc56ad76/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d1d1fe88-d4ae-43a3-8274-8abbdd3adc41) | 49  | 73  | 51  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.eaad496a-cc8c-4d3c-91e0-7613679d6e7b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/739de997708114c00159025b114f5103a5660610/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ee78122c-7dc5-4f3b-9fc0-bdd3f7473b36) | 35  | 93  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/739de997708114c00159025b114f5103a5660610/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/b4193fd4-ac18-4d25-b380-579239b52834) | 49  | 73  | 51  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.6c846e9e-38f1-48dc-b4b1-edef4c16899b?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/b38a81b6ab8238c8ed985c3f7096bbae42b224a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6272dcca-457c-4fa4-a54f-f4d4efcc8198) | 35  | 93  | 86  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/b38a81b6ab8238c8ed985c3f7096bbae42b224a5/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ec7787ea-47dc-4a8b-81bf-44cf63c4cf0f) | 49  | 73  | 51  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c9dc55b2-252d-4bd5-88c1-2997db82da50?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/c8dd5dddee189c2d8e1b621795919a01bc72397d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/d6075473-f40e-4e30-91b8-b267c0ae47b2) | 35  | 100  | 76  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/c8dd5dddee189c2d8e1b621795919a01bc72397d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f600a1ef-06e7-44ab-81d3-da79c2175f5f) | 48  | 73  | 50  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.e917a62d-dcf3-4db9-b158-d3a133aaad6c?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM! 👍

File: packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md [hunk 1]
```diff
@@ -206,13 +201,28 @@ The SDK communicates with an Okta server using the HTTPS protocol and requires a
...
+    * How to get your OIDC app object properties
+    * How to update your OIDC app to include the `urn:ietf:params:oauth:grant-type:jwt-bearer` grant type
+    * How to obtain a token with your OIDC app client ID
+
+    Fork this collection and add `url`, `apiKey`, `appId`, and `yourClientId` environment variables to run the example endpoints. The `PUT` method is a full property-replace operation, so you need to specify all required OIDC app properties, including any previous grant types. See [Create an API token](/docs/guides/create-an-api-token/main/) to obtain an `apiKey` from your org for testing purposes.
+
+* Alternatively, when you use the Admin Console to add or update a custom authenticator with an existing OIDC application, then the application automatically updates with the JWT Bearer grant type. See [Add a custom authenticator](#add-a-custom-authenticator).
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md]:
```suggestion
* Alternatively, when you use the Admin Console to add or update a custom authenticator with an existing OIDC application, the application automatically updates with the JWT Bearer grant type. See [Add a custom authenticator](#add-a-custom-authenticator).
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/d3bfe334a463df169a380099a8d4cfbd78f77531/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/070e9668-ff1b-47a5-acc6-cc27d184af5a) | 35  | 100  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/d3bfe334a463df169a380099a8d4cfbd78f77531/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ae157d85-23cd-4b5d-8f78-a7f3619d89e3) | 48  | 74  | 49  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.54d809c2-401d-4a88-8488-eacc5d24fdf9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by jane-kuang-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md [hunk 2]
```diff
@@ -206,13 +201,28 @@ The SDK communicates with an Okta server using the HTTPS protocol and requires a
...
 * Request pending push challenges
 * Enable and disable CIBA capability for the push authenticator enrollment
 * Update device tokens for push authenticator enrollment
 
-Usage example:
+To successfully obtain the maintenance token, you must first configure your Okta OIDC application to support the JWT Bearer grant type:
+
+* You can use the Apps API's [update application](/docs/reference/api/apps/#update-application) operation (`PUT /apps/${appId}`) to modify the `settings.oauthClient.grant_types` property array to include the JWT Bearer grant type, `urn:ietf:params:oauth:grant-type:jwt-bearer`.
+
+    Explore the [Configure and Use JWT Bearer Grant](https://god.gw.postman.com/run-collection/26510466-46beb74b-4755-4cf0-9847-845ccac1ccbd?action=collection%2Ffork&collection-url=entityId%3D26510466-46beb74b-4755-4cf0-9847-845ccac1ccbd%26entityType%3Dcollection%26workspaceId%3Daf55a245-1ac6-42d1-8af4-11e21e791e4e) Postman Collection for API examples of
```
Comment (isaacdurham-okta) [inline packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md]: Aren't we supposed to format lists like with a colon?

Explore the []() Postman Collection for examples of how to do the following:
* Get your OIDC...
* Update your OIDC
* Obtain....

File: packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md (hunk 1 above)
Comment (isaacdurham-okta) [reply packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md]: Match any changes here in the other topic.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/520cc484aba125e0124bdb48d9594056c4f870f6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/991ca448-9fbb-46dd-a45e-760de21d37ea) | 35  | 100  | 72  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/520cc484aba125e0124bdb48d9594056c4f870f6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/ede4f90f-16c7-4eb9-b20c-18a94fc9dff4) | 48  | 74  | 49  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.c1478782-a923-4661-bc8a-6f0462afff57?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/android/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/821bc3cf770d7808f92f66599475ab7e15058412/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fandroid%2Fsdksteps.md) | [82](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c0340fb5-edb2-42b3-87d0-9f72404b243a) | 35  | 100  | 71  | 100  |
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/ios/sdksteps.md](https://github.com/okta/okta-developer-docs/blob/821bc3cf770d7808f92f66599475ab7e15058412/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fauthenticators-custom-authenticator%2Fmain%2Fios%2Fsdksteps.md) | [75](https://okta.acrolinx.cloud/api/v1/checking/scorecards/392c1373-c647-4879-90d6-f9ed01260a5b) | 48  | 73  | 49  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.3e0a99ac-78ea-4b4c-afeb-a711785a25bd?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**